wiki/staking.md

```diff
@@ -0,0 +1,157 @@
+---
```

Comment:
Why is the staking page being introduced in a PR about image cleanup?

Reply:
Likely got modified by one of the cherry-picks and I mistakenly added it instead of deleting it.

I've removed the file.